.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: c
+dist: trusty
```

Comment:
This means that the resulting binaries should run on distributions from 2014 and later. It is unlikely to work on older distributions, but newer ones should be OK.